components/account/AccountHeader.vue

```diff
@@ -20,6 +20,7 @@ const relationship = $(useRelationship(account))
 
 const namedFields = ref<mastodon.v1.AccountField[]>([])
 const iconFields = ref<mastodon.v1.AccountField[]>([])
+const hasHeader = !account.header.endsWith('/original/missing.png')
```

Comment:
```suggestion
const hasHeader = $computed(() => !account.header.endsWith('/original/missing.png'))
```

How about using `$computed()` here?